docker/gitlab-docker-ci/verify.py

```diff
@@ -0,0 +1,18 @@
+import subprocess
+import sys
+
+def verify_python_version(version):
+    """ Verify if provided python version is correctly installed. """
+    try:
+        subprocess.run([f"/root/.pyenv/shims/python{version}", "-c", "import sys; print(sys.version)"], check=True, text=True, stdout=subprocess.PIPE)
```

Comment:
i think a better check would be to run pyenv list versions or something to check we can access them through pyenv